labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>The image is not showing up (at least for me).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Neither for me</p>

Reply:
<p>Hm, it is in my Dropbox, which might be a bad idea. Will put in github</p>